src/utils.py

```diff
@@ -55,55 +143,59 @@ def get_body(url: str, random_seed: float = None) -> str:
             response_successful = True
         else:
             print('response failure, retrying...')
-            time.sleep(random.random()*3)
+            time.sleep(random.random() * 3)
 
-    #clean up
+    # clean up
     if not response_successful:
-        raise UrlException
+        raise UrlException(" Sorry, we not able to down load this content at the moment.")
```

Comment:
Grammar police: "Sorry, we *are* not able to *download* this content at the moment."

Reply:
You are not wrong lol I'll update.